http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcAuthenticationMechanism.java

```diff
@@ -58,6 +60,7 @@ public String getMechanismName() {
 
     @Override
     public void evaluateRequest(HttpServerRequest request) throws HttpAuthenticationException {
+        log.debugf("evaluateRequest uri: " + request.getRequestURI().toString());
```

Comment:
```suggestion
        log.debugf("Evaluating request URI: [%s]", request.getRequestURI());
```

Reply:
changed

Reply:
There are more of those. I'm not sure which are meant to stay.

There are also several `debugf` where `debug` would be sufficient instead.